app/src/main/java/com/checkin/app/checkin/home/epoxy/NearbyRestaurantModelHolder.kt

```diff
@@ -0,0 +1,117 @@
+package com.checkin.app.checkin.home.epoxy
+
+import android.view.View
+import android.view.ViewGroup
+import android.widget.ImageView
+import android.widget.TextView
+import androidx.constraintlayout.widget.ConstraintLayout
+import butterknife.BindView
+import com.airbnb.epoxy.EpoxyAttribute
+import com.airbnb.epoxy.EpoxyModelClass
+import com.airbnb.epoxy.EpoxyModelWithHolder
+import com.checkin.app.checkin.R
+import com.checkin.app.checkin.home.model.NearbyRestaurantModel
+import com.checkin.app.checkin.misc.epoxy.BaseEpoxyHolder
+import com.checkin.app.checkin.restaurant.activities.openPublicRestaurantProfile
+import com.checkin.app.checkin.utility.Utils
+
+@EpoxyModelClass(layout = R.layout.item_home_restaurant_banner)
+abstract class NearbyRestaurantModelHolder : EpoxyModelWithHolder<NearbyRestaurantModelHolder.Holder>() {
+
+    @EpoxyAttribute
+    internal lateinit var restaurantModel: NearbyRestaurantModel
+
+    @EpoxyAttribute
+    var restaurantOpen: Boolean = false
```

Comment:
Booleans are named, with prefix of either "is" or "has".

Reply:
And technically, we don't need to expose this attribute since this property is inherent to **NearbyRestaurantModel**